api/swagger/swagger.yaml

```diff
@@ -122,6 +122,31 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /publicaions/{id}:
+    x-swagger-router-controller: publicaions
+    get:
+      tags:
+        - publicaions
+      description: Returns publicaion details
```

Comment:
There are a few `publicaion` typos. Please just grep for it across the codebase.